modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -482,6 +490,91 @@ import {TextInput} from '@workday/canvas-kit-react/text-input';
 
 🤖 The codemod will handle the change of `hasError` to `error` for you automatically.
 
+### Icons
+
+All icon components, `Svg`, `SystemIcon`, `SystemIconCircle`, `AccentIcon`, `AppletIcon`, `Graphic`,
+now use
+[Canvas Tokens](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs) and
+our
+[new styling utilities](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-modifiers#createstyles-api).
+The component now supports the `cs` prop, but otherwise the API has not changed. It should behave
+identically as it did in previous versions.
+
+The `styles` prop has been removed from icon components.
+
+🤖 The codemod will handle the change of `styles` to `cs` for you automatically.
+
+#### AccentIcon
+
+`AccentIconStyles` prop has been deprecated and will be merged with `AccentIconProps` in a future
+version. `accentIconStyles` has been deprecated and will be removed in a future version as a part of
+implementation of stencils and new tokens. Consider to use `accentIconStencil` to get styles or
+class name.
+
+#### AppletIcon
+
+`AppletIconStyles` prop has been deprecated and will be merged with `AppletIconProps` in a future
+version. `appletIconStyles` has been deprecated and will be removed in a future version as a part of
+implementation of stencils and new tokens. Consider to use `appletIconStencil` to get styles or
+class name.
+
+#### Graphic
+
+`GraphicStyles` prop has been deprecated and will be merged with `GraphicProps` in a future version.
+`graphicStyles` has been deprecated and will be removed in a future version as a part of
+implementation of stencils and new tokens. Consider to use `graphicStencil` to get styles or class
```

Comment:
I think we should give an example of how to get the className or styles, also not sure if this should be part of the PR? Are we ready to show the use of stencils and are we exporting them? cc @alanbsmith @NicholasBoll